web/frontend/src/components/input-controller/index.svelte

```diff
@@ -3,9 +3,10 @@
   import type { inputControllerApi } from '@viamrobotics/sdk';
   import Collapse from '@/lib/components/collapse.svelte';
 
+  type InputControllerEvent = (Omit<inputControllerApi.Event.AsObject, 'value'> & {value?: number});
```

Comment:
I noticed the responses from the status stream was not guaranteeing `value` to be defined. So This is overwriting that to match the responses.